src/templates/eth2.js

```diff
@@ -192,6 +193,7 @@ const components = {
   Eth2BeaconChainActions,
   Eth2ShardChainsList,
   Eth2DockingList,
+  CardContainer,
```

Comment:
This isn't needed on this template, at least not yet.

Reply:
hah, didn't realize this component was individually imported into the other eth2 pages like 'vision' page... so this dwindled down to just importing/exporting `CardContainer` in the static template, and fixing that one small `key` for the `##` header in the markdown file. 
Again, not getting banner issues at the moment 🤞